seqr/management/tests/reload_saved_variant_json_tests.py

```diff
@@ -1,11 +1,13 @@
 #-*- coding: utf-8 -*-
```

Comment:
you should rename this file to _2_3_test.py

Reply:
Okay. Changed.